app/src/actions/actions.js

```diff
@@ -1,8 +1,6 @@
 import * as ACTION_TYPES from './types';
 
-export const saveReactAppName = (name) => {
-    return {
-        type: ACTION_TYPES.SAVE_REACT_APP_NAME,
-        payload: name
-    };
-};
\ No newline at end of file
+export const saveReactAppName = name => ({
+    type: ACTION_TYPES.SAVE_REACT_APP_NAME,
+    payload: name,
+});
```

Comment:
We don't need this anymore

Reply:
@chimano actually I want to leave this to demonstrate how I want the connection with redux to be done for future store interactions.

Reply:
Fair enough